composer.json

```diff
@@ -23,7 +23,7 @@
         "sonata-project/datagrid-bundle": "^2.0",
         "sonata-project/doctrine-extensions": "^1.1",
         "symfony/config": "^2.8 || ^3.2 || ^4.0",
-        "symfony/dependency-injection": "^2.8 || ^3.2 || ^4.0",
+        "symfony/dependency-injection": "^2.8 || ^3.2 || ^4.0,!=4.4.0",
```

Comment:
Shouldn't be better to add `4.4.0` as conflicting version?

Reply:
IMO It's the same but you have all the constraints in the same place.

Reply:
I agree with that point. Just that using the "conflict" section seems more explicit or semantically correct to me (if in any case we would remove this dependency, but that will not happen because this package is abandoned).
:+1: